staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -22,6 +22,14 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+// Header keys used by the audit system.
+const (
+	// Header to hold the audit ID as the request is propagated through the serving hierarchy. The
+	// Audit-ID header should be set by the first server to receive the request (e.g. the federation
+	// server or kube-aggregator).
+	HeaderAuditID = "Audit-ID"
```

Comment:
Let's move the discussion of whether audit IDs should be 1 or many to here.

@deads2k - I think part of my reservation around multiple audit IDs is I don't understand what threat the audit trace would help to capture. Can you elaborate?